sources/platform/actors/running/actor_standby.md

```diff
@@ -31,6 +31,14 @@ If you're using an Actor built by someone else, see its Information tab to find
 
 Generally speaking, Actors in Standby mode behave as standard HTTP servers. You can use any of the existing [HTTP request methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) like GET, POST, PUT, DELETE, etc. You can pass the input via [HTTP request query string](https://en.wikipedia.org/wiki/Query_string) or via [HTTP request body](https://developer.mozilla.org/en-US/docs/Web/HTTP/Messages#body).
 
+## How do I authenticate my requests
+
+The authentication of requests to Actor Standby works the same as [with the Apify API](../../integrations/programming/api.md).
+To authenticate a request, add your [API token](../../integrations/programming/api.md#api-token) either:
+
+- to your request's `Authorization` header as `Bearer <token>` (e.g. `Authorization: Bearer my_apify_token`). This is the recommended option, as it will prevent the token being logged in server logs.
+- as the `token` parameter to your request's URL (e.g. `https://rag-web-browser.apify.actor/search?query=apify&token=my_apify_token`). This is useful if you can't modify the request headers.
+
```

Comment:
Alternatively maybe this approach will be better?

```suggestion
## How do I authenticate my requests

To authenticate requests to Actor Standby, follow the same process as [authenticating requests to the Apify API](../../integrations/programming/api.md)
You can provide your  [API token](../../integrations/programming/api.md#api-token) in one of two ways:

1. _Recommended_: Include the token in the `Authorization` header of your request as `Bearer<token>`. This approach is recommended because it prevents your token from being logged in server logs.

	```
	Authorization: Bearer my_apify_tokin
	```

2. Alternatively, you can append the token as a query parameter named `token` to the request URL:
	
	```text
	https://rag-web-browser.apify.actor/search?query=apify&token=my_apify_token
	```
This method can be useful if you cannot modify the request header
```

Reply:
check the suggestion above, usually tabbing within the list should allow you to create bigger code examples, we used this approach in [here](https://docs.apify.com/platform/actors/development/actor-definition/source-code#example-setup)

Reply:
Cool! I integrated the suggestion, please take another look.